lib/iex/lib/iex/autocomplete.ex

```diff
@@ -229,6 +229,9 @@ defmodule IEx.Autocomplete do
       %{kind: :module, type: :elixir, name: Enum.at(parts, depth - 1)}
     end
     |> Enum.uniq
+    |> Enum.filter(fn(%{name: name}) ->
+      !match?(":" <> _, inspect(String.to_atom("Elixir." <> name)))
```

Comment:
True. Only I'm not so sure it makes it more readable. But it's more efficient indeed.